BUILD.bazel

```diff
@@ -79,19 +79,19 @@ gazelle(
 
 alias(
     name = "gobin",
-    actual = "@go_sdk//:bin/go",
+    actual = "@rules_go//go",
     visibility = ["//visibility:public"],
 )
 
-# Builds python dependencies
+# Builds python dependencies. To update the lockfile:
+# $ bazel run //:python-requirements.update
 compile_pip_requirements(
     name = "python-requirements",
-    timeout = "moderate",
-    requirements_in = "requirements.in",
+    src = "requirements.in",
+    # NOTE: the requirements.txt reports the system python's version, not
+    # the one specified in MODULE.bazel.
+    # https://rules-python.readthedocs.io/en/latest/pypi-dependencies.html#pip-rules
     requirements_txt = "requirements.txt",
-    tags = [
-        "requires-network",
-    ],
```

Comment:
Why did you remove this? I believe it does require the network as it needs to check the latest version.

Reply:
It only needs network when you `bazel run` it AFAICT (`bazel run //:python-requirements.update`) . For regular builds it definitely shouldn't need network

Reply:
Okay now I'm not sure anymore... I know that `compile_pip_requirements` generates a test called `python-requirements_test` that validates if the requirements are still up to date and that failed in the past when `requires-network` was not specified. But it looks like it succeeded on your PR, so maybe that is not an issue anymore.